package.json

```diff
@@ -28,18 +29,20 @@
     "LICENSE.md"
   ],
   "devDependencies": {
-    "@babel/core": "^7.23.7",
-    "@babel/eslint-parser": "^7.23.3",
-    "@babel/preset-env": "^7.23.8",
-    "@playcanvas/eslint-config": "^1.7.4",
-    "@playcanvas/observer": "1.3.2",
+    "@babel/core": "^7.26.0",
+    "@babel/eslint-parser": "^7.25.9",
+    "@babel/preset-env": "^7.26.0",
```

Comment:
Since the engine today migrated to SWC, we should gradually roll this out to other repos. Can be done before or after TS port.